lib/python/qmk/keyboard.py

```diff
@@ -30,6 +30,28 @@
         "h": "_",
     },
 }
+ENC_DRAWING_CHARACTERS = {
+    "unicode": {
+        "tl": "🮣",
+        "tr": "🮢",
+        "bl": "🮡",
+        "br": "🮠",
+        "vl": "│",
+        "vr": "│",
+        "v": "│",
+        "h": "─",
```

Comment:
how about these?
```suggestion
        "tl": "╭",
        "tr": "╮",
        "bl": "╰",
        "br": "╯",
        "vl": "│",
        "vr": "│",
        "v": "│",
        "h": "─",
```

Reply:
Already tried, visually they are not different enough to the regular key.

Reply:
Windows being windows, ive gone with the rounded corners and some arrows. Someone else can improve this later if they wish.